packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java

```diff
@@ -58,11 +92,19 @@ public static RestRuleModel createRuleModel(String name, List<RestActionBodyExec
      *
      * @return The created action model.
      */
-    public static RestActionBodyExecTemplateModel createActionModel()
+    public static RestActionBodyExecTemplateModel createDefaultActionModel()
     {
         RestActionBodyExecTemplateModel restActionModel = new RestActionBodyExecTemplateModel();
         restActionModel.setActionDefinitionId("add-features");
         restActionModel.setParams(Map.of("aspect-name", "{http://www.alfresco.org/model/audio/1.0}audio", "actionContext", "rule"));
         return restActionModel;
     }
+
+    public static RestCompositeConditionDefinitionModel createEmptyConditionModel()
+    {
+        RestCompositeConditionDefinitionModel conditions = new RestCompositeConditionDefinitionModel();
+        conditions.setInverted(false);
+        conditions.setBooleanMode("and");
```

Comment:
These two fields should be optional (with a default), so we should be able to leave them as `null`. If we can't do that at the moment then let's make a separate change to fix it.